index.html

```diff
@@ -0,0 +1,291 @@
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-1.png"
+                  alt="company-1"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-2.png"
+                  alt="company-2"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-3.png"
+                  alt="company-3"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-4.png"
+                  alt="company-4"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-5.png"
+                  alt="company-5"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- end sectionfour -->
+    </main>
+    <!-- end main -->
+    <!-- start footer -->
+    <footer class="footer" id="footer">
+      <div class="container">
+        <div class="footer__container__parfoot">
+          <div class="footer__container__parfoot__foot">
+            <div class="footer__container__parfoot__foot__responsive">
+              <img
+                src="static/imgs/company-1.png"
+                alt=""
+                class="responsive footer__container__parfoot__foot__responsive__respo"
+              />
+            </div>
+          </div>
+          <div class="footer__container__parfoot__foot">
```

Comment:
- [ ]  As the previous reviewer suggested, your image is not visible, and seemed that you've not changed anything to make it look clear. I would recommend you use blackness logo.
![image](https://user-images.githubusercontent.com/43877689/197214543-4b823afc-4f97-4fba-8cb2-219d51414712.png)